### DISCUSSION TITLE
Variable not working in blade view with setConfigurableAreas

### DISCUSSION THREAD

**Discussion by luizsilva-dev** [rappasoft/laravel-livewire-tables#1182] [General]
I am trying to display a variable in the file being called by setConfigurableAreas.
The wire:model and wire:click functions work fine. But I can not display the variable in blade view.
The variable is public, but in the blade it returns "Undefined variable $clientNewError"

public $clientNewError = 'test';

**Comment by luizsilva-dev** [comment]
I tried using customView with the same blade view and it worked fine. But I need it to be in "toolbar-right-start", so I need to use setConfigurableAreas.

Any suggestion?

**Comment by lrljoe** [comment]
In your blade, use 
```php
$this->clientNewError
```

Or see

https://rappasoft.com/docs/laravel-livewire-tables/v2/datatable/configurable-areas#content-passing-parameters
For passing parameters
